snapmaker/src/module/toolhead_laser.cpp

```diff
@@ -294,15 +300,21 @@ ErrCode ToolHeadLaser::SetFocus(SSTP_Event_t &event) {
 
   buff[0] = (uint8_t)(focus >> 8);
   buff[1] = (uint8_t)(focus);
+  if (rotaryModule.status() != ROTATE_OFFLINE) {
+    buff[2] = 1;
+  } else {
+    buff[2] = 0;
+  }
 
   cmd.id     = MODULE_FUNC_SET_LASER_FOCUS;
   cmd.data   = buff;
-  cmd.length = 2;
+  cmd.length = 3;
```

Comment:
useing sizeof data would be better

Reply:
I don't think so, sometimes the length of buffer array will be larger then the used length